plugins/pod-shell.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha1
+kind: Plugin
+metadata:
+  name: pod-shell
+spec:
+  platforms:
+  - head: https://github.com/danisla/kubefunc/archive/v1.0.1.zip
+    sha256: c548f94d8efd1fb8aa35a54b7e5cdb64ae012447122d9704f3cee4163bbbe9c3
+    uri: https://github.com/danisla/kubefunc/archive/v1.0.1.zip
+    files:
+    - from: "/*/pod-shell/*"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+  version: "v1.0.1"
+  shortDescription: Run shell command in pod
```

Comment:
how's the use case for this look like? (trying to see what it adds over kubectl exec). do you mind providing a long description field with some examples for when the shortDescription isn't explanatory enough like this?

Reply:
This is more convenient because it shows you a list of pods that you choose from interactively and tells you if there are multiple pods. Again, super convenient for daily stuff I find my self repeating.

Reply:
That makes sense, we should get this in.